schema_editor/app/scripts/views/login/auth-controller.js

```diff
@@ -0,0 +1,47 @@
+(function() {
+    'use strict';
+
+    /**
+     * @ngInject
+     */
+    function AuthController ($scope, $state, $window, AuthService) {
+
+        $scope.auth = {};
+
+        $scope.alerts = [];
+        $scope.addAlert = function(alertObject) {
+            $scope.alerts.push(alertObject);
+        };
+        $scope.closeAlert = function(index) {
+            $scope.alerts.splice(index, 1);
+        };
+
+        $scope.authenticate = function() {
+            $scope.alerts = [];
+            $scope.authenticated = AuthService.authenticate($scope.auth);
+            $scope.authenticated.then(function(result) {
+                if (result.isAuthenticated) {
+                    // redirect to editor main page, with a full page reload,
+                    // to pick up the newly set credentials
+                    $window.location.href = '/editor/';
```

Comment:
This is only the url to editor main page on the built app. In development on port 7001, this url won't exist.